ArduPlane/GCS_Mavlink.cpp

```diff
@@ -372,6 +372,26 @@ void Plane::send_nav_controller_output(mavlink_channel_t chan)
         nav_controller->crosstrack_error());
 }
 
+void Plane::send_position_target_global_int(mavlink_channel_t chan)
+{
+    mavlink_msg_position_target_global_int_send(
+        chan,
+        AP_HAL::millis(), // time_boot_ms
+        MAV_FRAME_GLOBAL_INT, // targets are always global altitude
+        0xFFF8, // ignore everything except the x/y/z components
+        next_WP_loc.lat, // latitude as 1e7
+        next_WP_loc.lng, // longitude as 1e7
+        next_WP_loc.alt / 100.0f, // altitude is sent as a float
```

Comment:
Has anyone done any metrics comparing float divide versus multiply on pixhawk? I always cringe when I see /100 instead of *0.01